stages/org.osbuild.oscap.autotailor.meta.json

```diff
@@ -95,7 +95,19 @@
             },
             "description": "The variables to override in the base OpenSCAP profile"
           }
-        }
+        },
+        "oneOf": [
+          {
+            "required": [
+              "new_profile"
+            ]
+          },
+          {
+            "required": [
+              "tailored_profile_id"
+            ]
+          }
+        ]
```

Comment:
How does this interact with the `required` block above (L39)?  Does it override it or add to it?

Reply:
It adds to it, so the first block is shared. If it's clearer I could just have `oneOf` with all the required fields for each case and get rid of the first `required` block?

Reply:
I strongly prefer `required` being defined only once (that includes also case when it is defined multiple times, but inside `oneOf`) and in one place...

Reply:
I think it is clearer, so I went ahead and changed it.